ccd-definition/CaseEventToFields/actionCMO.json

```diff
@@ -3,14 +3,53 @@
     "LiveFrom": "01/01/2017",
     "CaseTypeID": "CARE_SUPERVISION_EPO",
     "CaseEventID": "actionCMO",
-    "CaseFieldID": "allPartiesLabelCMO",
+    "CaseFieldID": "cmoToAction",
     "PageFieldDisplayOrder": 1,
     "DisplayContext": "READONLY",
-    "PageID": "allPartiesDirections",
+    "PageID": "actionCMOPlaceholder",
+    "PageLabel": " ",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "FieldShowCondition": "actionCMOPlaceholderHeading=\"DO_NOT_SHOW\""
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "CaseTypeID": "CARE_SUPERVISION_EPO",
+    "CaseEventID": "actionCMO",
+    "CaseFieldID": "actionCMOPlaceholderHeading",
+    "PageFieldDisplayOrder": 2,
+    "DisplayContext": "READONLY",
+    "PageID": "actionCMOPlaceholder",
+    "PageLabel": " ",
+    "PageDisplayOrder": 1,
+    "PageColumnNumber": 1,
+    "PageShowCondition": "cmoToAction!=\"*\" OR cmoToAction.status!=\"JUDE_REVIEW\""
```

Comment:
if this (`cmoToAction!=\"*\"`) is doing what I think it is doing and checking for null then you can do `cmoToAction=\"\"` instead. To check against null with CCD you compare to an empty string...

Reply:
james has changed this now to cmoToAction=\"\" so resolving